### ISSUE TITLE
Support metalinter

### ISSUE THREAD

**Issue by santagada** [joefitzgerald/go-plus#280]
go metalinter can run a bunch of linters in parallel, I guess it would be a nice addition to run it on save instead of only golint

repo for metalinter:
https://github.com/alecthomas/gometalinter

**Comment by zmb3** [comment]
Duplicate of https://github.com/joefitzgerald/go-plus/issues/213
